fast-cart/src/components/Navbar.js

```diff
@@ -0,0 +1,41 @@
+import { useMemo } from "react";
+import { Link } from "react-router-dom";
+import { useShopping } from "../context/ShoppingListContext";
+
+const Navbar = () => {
+
+    const shoppingList = useShopping();
+
+    const totalItems = useMemo(()=>{
+        var number = 0;
+        for(var item in shoppingList){
```

Comment:
this will work fine, but for these cases we can use **reduce** of array also, which looks more cleaner. you can read about it it will be very interesting